### PR TITLE
[MM-33535] Mattermost apps deployment components

### PR DESCRIPTION
**Pull request by stylianosrigas**
#### Summary
- Mattermost apps cloud deployer components
- Infrastructure components needed by Mattermost apps bundle functions (IAM, SG, etc.)

#### Ticket Link
- https://mattermost.atlassian.net/browse/MM-33535

### CODE REVIEW

File: terraform/aws/modules/mattermost-apps-cloud-deployer/mattermost_apps_cloud_deployer_user_permissions.tf [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:ListBucket",
+                "s3:GetBucketLocation"
+            ],
+            "Resource": [
+                "arn:aws:s3:::${var.static_bucket}",
+                "arn:aws:s3:::${var.terraform_state_bucket}",
+                "arn:aws:s3:::terraform-cloud-monitoring-state-bucket-test"
```
Comment (stafot) [inline terraform/aws/modules/mattermost-apps-cloud-deployer/mattermost_apps_cloud_deployer_user_permissions.tf:26]: @stylianosrigas shouldn't this also be variable?

**Review by stafot — APPROVED** [review]
LGTM. A minor comment only.

File: terraform/aws/modules/mattermost-apps-cloud-deployer/mattermost_apps_cloud_deployer_user_permissions.tf (hunk 1 above)
Comment (stylianosrigas) [reply terraform/aws/modules/mattermost-apps-cloud-deployer/mattermost_apps_cloud_deployer_user_permissions.tf]: You are right, thanks a lot. Fixed ;)

**Review by stafot — APPROVED** [review]

**Review by angeloskyratzakos — APPROVED** [review]
LGTM

**Pull request merged**